evap/student/templates/student_vote.html

```diff
@@ -14,8 +14,8 @@
 
 {% block content %}
     {{ block.super }}
-    {% if errors %}
-        <div class="alert alert-danger" role="alert">{% blocktrans %}Please make sure to vote for all contributors or click on "I can't give feedback" to skip a person.{% endblocktrans %}</div>
+    {% if is_bound %}
```

Comment:
alright then please rename that variable to contributor_errors.

still, i'd like something more explicit than is_bound here. maybe look whether there are any errors in the general contribution, and then do "if general_contribution_errors or contributor_errors" here or something like this.